chips/nrf52/src/aes.rs

```diff
@@ -0,0 +1,229 @@
+//! AES128-CTR Driver
+//!
+//! Provides a simple driver for userspace applications to encrypt and decrypt messages
+//! using aes128-ctr mode on top of aes128-ecb
+//!
+//! The initial counter configured according to the counter received from the user application.
+//! The capsule is invoked as follows:
+//!     - the key has been configured
```

Comment:
Need an empty comment line before this list for the markdown to render correctly.